### DISCUSSION TITLE
how can i sign in google

### DISCUSSION THREAD

**Discussion by YunySun** [nextauthjs/next-auth#10984] [Help]
I followed the documentation and added the Google clientId and clientSecret, but after redirecting to the login page, it shows a service error.
![image](https://github.com/nextauthjs/next-auth/assets/48754120/6fe2e404-1596-4789-b582-7eef6c88f98b)
![image](https://github.com/nextauthjs/next-auth/assets/48754120/42e3a532-28cf-48df-b33e-2f69471a7608)
![image](https://github.com/nextauthjs/next-auth/assets/48754120/68212fbf-5e21-4c60-bbaf-7928e5aa5e16)
![image](https://github.com/nextauthjs/next-auth/assets/48754120/ce55b57d-1a7e-4a44-bb01-ae9ef3f9ea61)
And the GitHub login works without any issues.
system: window 11
node: v20.13.1
next: "latest"
next-auth: "beta"

**Comment by k-taro56** [comment]
Hi,

I could not confirm that the latest `next-auth (5.0.0-beta.18
)` source code, I could not confirm that such an error is displayed.
Could you please provide me with the detailed version?

Thank you.
